pyinteraph/path_analysis.py

```diff
@@ -7,8 +7,7 @@
 import re
 import matplotlib.pyplot as plt
 import seaborn as sns
-from matplotlib.colors import ListedColormap
-from matplotlib.colors import LinearSegmentedColormap
+import matplotlib.colors as col
```

Comment:
col is a dangerous abbreviation, since it can mean many different things and can easily cause name clashes. You can use ´mplcolors´ or something similar to be clearer.